dev/create-release/release-build.sh

```diff
@@ -186,6 +186,9 @@ if [[ $SPARK_VERSION < "3.2" ]]; then
 fi
 
 PUBLISH_SCALA_2_12=1
+if [[ $SPARK_VERSION < "4.0" ]]; then
+  PUBLISH_SCALA_2_12=0
+fi
```

Comment:
It seems PUBLISH_SCALA_2_12 and PUBLISH_SCALA_2_13 will be 0, if `SPARK_VERSION < "3.2"`.